.github/workflows/build.yaml

```diff
@@ -176,16 +176,22 @@ jobs:
           # seen before, so numerous tests don't know about it.
           [[ "$arch" == "MINGW6" ]] && arch=CYGWIN
           export AUTOBUILD="$(which autobuild)"
-          # Build with a tag like "Second_Life_Project_Shiny#abcdef0" to get a
-          # viewer channel "Second Life Project Shiny" (ignoring "#hash",
-          # needed to disambiguate tags).
-          if [[ "$GITHUB_REF_TYPE" == "tag" && "${GITHUB_REF_NAME:0:12}" == "Second_Life_" ]]
-          then viewer_channel="${GITHUB_REF_NAME%#*}"
-               export viewer_channel="${viewer_channel//_/ }"
-          else export viewer_channel="Second Life Test"
+
+          # determine the viewer channel from the branch name
+          branch=$AUTOBUILD_VCS_BRANCH
+          echo "branch=$branch" >> "$GITHUB_OUTPUT"
```

Comment:
This doesn't need to be put to `"$GITHUB_OUTPUT"` (thus creating `${{ steps.build.outputs.branch }}`) because it's already there as `${{ steps.which-branch.outputs.branch }}` -- that's how we set `$AUTOBUILD_VCS_BRANCH` at [line 108](https://github.com/secondlife/viewer/blob/7f23f160d7520034afd8c3be8a184eac1716227a/.github/workflows/build.yaml#L108).